### ISSUE TITLE
Mapping options

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#933]
![image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/20424460/ea9383a0-ca3e-443e-b124-d1d3bf5b3021)

**Comment by FLO-2DJJ** [comment]
@FLO-2DKaren 
What's this issue all about? A reminder? Future development? Where does the screenshot come from?

**Comment by FLO-2DKaren** [comment]
@FLO-2DJJ
Nothing yet.  Its just a reminder for me when we start mapping development.

**Comment by rpachaly** [comment]
We could transfer this issue to FLO-2D MapCrafter I think.

**Comment by FLO-2DKaren** [comment]
Yes I think that would be a good idea.  That was my original plan was to add it to our mapping tools.
